app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/MainFragment.kt

```diff
@@ -173,4 +169,8 @@ class MainFragment : Fragment() {
         return view
     }
 
+    companion object {
+        fun newInstance() = MainFragment()
+    }
+
```

Comment:
Companion objects should exist at the bottom of a class based on coding standards.

Reply:
Yes.  Good suggestion.